content/shmem_malloc_hints.tex

```diff
@@ -44,6 +44,17 @@
     The user is also responsible for calling these routines with identical argument(s) on all
     \acp{PE}; if differing \VAR{size}, or \VAR{hints} arguments are used, the behavior of the call
     and any subsequent \openshmem calls is undefined.
+
+	Calling the routine \FUNC{shmem\_realloc(void *ptr, size\_t size)} on the
```

Comment:
Twice here it says "the ptr allocated", I think it would be better to say "a ptr allocated".